### ISSUE TITLE
Install weechat and expose the port

### ISSUE THREAD

**Issue by decentral1se** [jodeko/glowing_bear_ynh#9]
This could probably be prompted for (if you want full control, you probably already installed weechat yourself and configured it) but would be nice for the ones who just want to get moving fast. Install weechat, a special weechat user and also expose the correct port. Then it is also easy to get moving with Weechat Android faster too.

**Comment by alexAubin** [comment]
I completely agree and to me not installing weechat and everything else necessary for the ap p to just work out of the box is in contradiction with the spirit of YunoHost ... In this context, installing weechat requires having to ssh and to know how to install and configure that package, which is mean non-techy users (or even techy users, like my flatmate) are not going to be frustrated that this app ain't "working" despite it being flagged as level 7 ...

I'm considering flagging this app as low quality (using [this mechanism](https://github.com/YunoHost/package_linter/pull/77)) until this is fixed...

**Comment by decentral1se** [comment]
Fair enough! Agree with reasoning. 

I won't be getting back to this anytime soon so it probably also requires a mainainter...

**Comment by carrabelloy** [comment]
I installed the whole thing via yunohost. But then I want to set up the system and that doesn't work. I wanted to play the encryption on it that didn't work. Okay then I thought I'd do it without encryption for now and see if the retrofitting can be done, but that doesn't work either without encryption. Can someone please help me out here. The same is true for the matrix too. Apparently I always can't get a connection to the server. It would be nice if someone here can help me with a few tips. I have to work with translators. I hope everyone understands the thank you in advance.

![grafik](https://user-images.githubusercontent.com/1060591/119662502-c142fd80-be31-11eb-89a3-ad8c95413ccc.png)
